### ISSUE TITLE
Schema does not include documentation of measurement quality

### ISSUE THREAD

**Issue by tomkralidis** [wmo-im/wmdr#13]
(moved from https://github.com/wmo-cop/wmo-oscar/issues/22)

Schema does not include documentation of measurement quality (WMDS elements 8-01 till 8-05). Will be considered for inclusion in schema version 2.0

**Comment by joergklausen** [comment]
Some of this is addressed with the uncertainty-related attributes of Equipment, as well as in the Reporting ... but we're only scratching the surface. Also, the current handling of this is not using any international standard such as ISO 19157.

@tomkralidis A broader consultation in ET Metadata may be needed, what do you think?

**Comment by tomkralidis** [comment]
@joergklausen makes sense. Have you considered `gmd:dataQualityInfo` in 19115 proper (already included in WMDR), which would be lighter than 19157 (though scoped for datasets)?

**Comment by atverm** [comment]
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
@prefix dct:  <http://purl.org/dc/terms/> .
@prefix owl:  <http://www.w3.org/2002/07/owl#> .
@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .

@base <https://envri.eu/vocab/quality/> .

<https://envri.eu/vocab/quality/>
  a owl:Ontology ;
  dct:title "ENVRI Quality Vocabulary"@en ;
  owl:versionInfo "1.0.0"^^xsd:string .

### Concept Schemes

<dimension/>
  a skos:ConceptScheme ;
  dct:title "ENVRI Data Quality Dimensions"@en .

<metric/>
  a skos:ConceptScheme ;
  dct:title "ENVRI Data Quality Metrics"@en .

<qcflag/>
  a skos:ConceptScheme ;
  dct:title "ENVRI QC Flags"@en .

<application/>
  a skos:ConceptScheme ;
  dct:title "ENVRI Application Areas"@en .

### Dimensions (seed)

<dimension/completeness>
  a skos:Concept ;
  skos:inScheme <dimension/> ;
  skos:prefLabel "Completeness"@en ;
  skos:definition "Degree to which expected data are present."@en .

<dimension/timeliness>
  a skos:Concept ;
  skos:inScheme <dimension/> ;
  skos:prefLabel "Timeliness"@en ;
  skos:definition "Degree to which data are up to date for an intended use."@en .

<dimension/accuracy>
  a skos:Concept ;
  skos:inScheme <dimension/> ;
  skos:prefLabel "Accuracy"@en ;
  skos:definition "Closeness of observations to the true value (as defined by method/traceability)."@en .

<dimension/consistency>
  a skos:Concept ;
  skos:inScheme <dimension/> ;
  skos:prefLabel "Consistency"@en ;
  skos:definition "Degree to which data are free from contradictions within a dataset and across related datasets."@en .

### Metrics (seed)

<metric/completenessRatio/v1>
  a skos:Concept ;
  skos:inScheme <metric/> ;
  skos:prefLabel "Completeness Ratio (v1)"@en ;
  skos:definition "observed_count / expected_count"@en .

<metric/dataLatencyHours/v1>
  a skos:Concept ;
  skos:inScheme <metric/> ;
  skos:prefLabel "Data Latency (hours) (v1)"@en ;
  skos:definition "Time difference in hours between observation time and publication time."@en .

### QC Flags (seed)

<qcflag/good>
  a skos:Concept ;
  skos:inScheme <qcflag/> ;
  skos:prefLabel "Good"@en ;
  skos:definition "Value passed QC checks."@en .

<qcflag/suspect>
  a skos:Concept ;
  skos:inScheme <qcflag/> ;
  skos:prefLabel "Suspect"@en ;
  skos:definition "Value is potentially problematic; use with caution."@en .

<qcflag/bad>
  a skos:Concept ;
  skos:inScheme <qcflag/> ;
  skos:prefLabel "Bad"@en ;
  skos:definition "Value failed QC; do not use for analysis."@en .

### Application areas (seed)

<application/climateMonitoring>
  a skos:Concept ;
  skos:inScheme <application/> ;
  skos:prefLabel "Climate Monitoring"@en .

<application/forecastingNRT>
  a skos:Concept ;
  skos:inScheme <application/> ;
  skos:prefLabel "Forecasting / Near-Real-Time Operations"@en .


and then an example:

{
  "@context": {
    "skos": "http://www.w3.org/2004/02/skos/core#",
    "dct": "http://purl.org/dc/terms/",
    "envri": "https://envri.eu/vocab/quality/",
    "xsd": "http://www.w3.org/2001/XMLSchema#"
  },
  "@id": "https://envri.eu/vocab/quality/profile/forecastingNRT/v1",
  "@type": "envri:QualityWeightProfile",
  "skos:prefLabel": "Forecasting / NRT Weight Profile v1" ,
  "dct:conformsTo": "https://envri.eu/spec/quality-profile/1.0",
  "envri:applicationArea": "https://envri.eu/vocab/quality/application/forecastingNRT",
  "envri:weights": [
    { "envri:dimension": "https://envri.eu/vocab/quality/dimension/timeliness",   "envri:weightValue": 0.50 },
    { "envri:dimension": "https://envri.eu/vocab/quality/dimension/completeness", "envri:weightValue": 0.20 },
    { "envri:dimension": "https://envri.eu/vocab/quality/dimension/accuracy",     "envri:weightValue": 0.20 },
    { "envri:dimension": "https://envri.eu/vocab/quality/dimension/consistency",  "envri:weightValue": 0.10 }
  ],
  "envri:recommendedThresholds": {
    "envri:minimumAcceptableScore": 0.80,
    "envri:highQualityScore": 0.95
  }
}
